packages/@okta/vuepress-site/docs/guides/pwd-optional-sign-in-email/main/react/integrationsteps.md

```diff
@@ -0,0 +1,130 @@
+### 1: The user navigates to the sign-in page
+
+When the user navigates to your app's sign-in page, call `OktaAuth.idx.start()` to start the sign-in flow.
+
+```javascript
+const { oktaAuth } = useOktaAuth();
+const startFlow = useCallback(async () => {
+  newTransaction = await oktaAuth.idx.start();
+}, [oktaAuth, flow, setTransaction]);
+```
+
+The method returns an `IdxTransaction` object containing field metadata that can be used to create a sign-in page dynamically.
+
+```json
+{
+   "status":"PENDING",
+   "nextStep":{
+      "name":"identify",
+      "inputs":[
+         {
+            "name": "username",
+            "label": "Username",
+            "required": true
+         }
+         ...
+      ]
+   },
+}
+```
+
+Display &mdash; either statically or dynamically using `IdxTransaction` &mdash; an input field for the user's username.
+
+<div class="half border">
+
+![Screenshot showing a sign-in page with a username field and a Next button.](/img/pwd-optional/pwd-optional-sign-in-page.png)
+
+</div>
+
+### 2: The user submits their username
+
+When the user submits their account details, create an object with a `username` property and assign it the value entered by the user.
+
+```javascript
+const inputValues = {
+  username: "johndoe@email.com",
+ };
+```
+
+Send this new object to `OktaAuth.idx.proceed()`.
```

Comment:
Sending a value to a method doesn't make sense.

```suggestion
Call `OktaAuth.idx.proceed()` passing in this new object as a parameter.
```

Reply:
Updated. But curiously, why doesn't "send ..." make sense?